### ISSUE TITLE
Important: Add CONTRIBUTING.md📝

### ISSUE THREAD

**Issue by deepanshu-rawat6** [upes-open/OSoC-bio.lnk#19]

**Comment by deepanshu-rawat6** [comment]
Here you go @karthiknadar1204 , make sure to create a descriptive ```CONTRIBUTING.MD```. If in doubts refer to this comment [here](https://github.com/upes-open/OSoC-bio.lnk/issues/17#issuecomment-1625638624)

**Comment by deepanshu-rawat6** [comment]
Hey @karthiknadar1204 , please leave a comment such that I can assign you this issue.

**Comment by karthiknadar1204** [comment]
I would like to work on this issue under ossoc'23

**Comment by karthiknadar1204** [comment]
Will this be counted for points,please reconfirm that once @deepanshu-rawat6

**Comment by deepanshu-rawat6** [comment]
Yes of course, you'll be rewarded points for completion of this issue

**Comment by karthiknadar1204** [comment]
ohk, will do, I was initially told that documentation issues will not be counted for points

**Comment by deepanshu-rawat6** [comment]
No, for add a CONTRIBUTING.md, it'll be an easy challenge, so you will get 10 points
